### PR TITLE
Document Update

### PR DESCRIPTION
**Pull request by NiteshKoushik**
Where to were

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/installation/upgrading.md [hunk 1]
```diff
@@ -274,6 +274,6 @@ The following alert notifiers have been updated to support storing their sensiti
 
 For existing alert notification channels, there is no automatic migration of storing sensitive settings encrypted, and they will continue to work as before. Migration must be done manually. Opening a configured alert notification channel in the UI and saving it will store sensitive settings encrypted and at the same time reset the historic unencrypted setting of that alert notification channel in the database.
 
-> Please note that when migrating a notification channel and later downgrading Grafana to an earlier version, the notification channel will not be able to read stored sensitive settings and by that not function as expected.
+> Please note that when migrating a notification channel and later downgrading Grafana to an earlier version, the notification channel will not be able to read stored sensitive settings and by that it does not function as expected.
```
Comment (aknuds1) [inline docs/sources/installation/upgrading.md:277]:
```suggestion
> Please note that when migrating a notification channel and later downgrading Grafana to an earlier version, the notification channel will not be able to read stored sensitive settings and as a result not function as expected.
```

File: docs/sources/installation/upgrading.md [hunk 2]
```diff
@@ -252,7 +252,7 @@ Grafana now requires backend plugins to be signed. If a backend plugin is not si
...
+Starting from Grafana v7.0.0, the cookie path does not include the trailing slash if Grafana is served from a subpath in order to align with [RFC 6265](https://tools.ietf.org/html/rfc6265#section-5.1.4). However, stale session cookies (set before the upgrade) can result in unsuccessful logins because they cannot be deleted during the standard login phase due to the changed cookie path. Therefore users experiencing login problems are advised to manually delete old session cookies, or administrators can fix this for all users by changing the [`login_cookie_name`]({{< relref "../administration/#login-cookie-name" >}}), so the old cookie would get ignored.
```
Comment (aknuds1) [inline docs/sources/installation/upgrading.md:255]: I would revert this change, since it's completely subjective (i.e., the original wording is just as correct).

**Review by aknuds1 — CHANGES_REQUESTED** [review]

File: docs/sources/installation/upgrading.md [hunk 3]
```diff
@@ -189,8 +189,8 @@ this new setting.
 ### Session storage is no longer used
 
 In 6.2 we completely removed the backend session storage since we replaced the previous login session implementation with an auth token.
-If you are using Auth proxy with LDAP an shared cached is used in Grafana so you might want configure [remote_cache] instead. If not
-Grafana will fallback to using the database as an shared cache.
+If you are using Auth proxy with LDAP a shared cached is used in Grafana so you might want configure [remote_cache] instead. If not
+Grafana will fallback to using the database as a shared cache.
```
Comment (aknuds1) [inline docs/sources/installation/upgrading.md:193]:
```suggestion
If you are using Auth proxy with LDAP, a shared cached is used in Grafana, so you might want to configure [remote_cache] instead. If not,
Grafana will fallback to using the database as a shared cache.
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]

File: docs/sources/installation/upgrading.md (hunk 2 above)
Comment (NiteshKoushik) [reply docs/sources/installation/upgrading.md:255]: I feel, both looks same and no changes present.

Comment (aknuds1) [reply docs/sources/installation/upgrading.md:255]: There is a change from "can not" to "cannot".

**Comment by NiteshKoushik** [comment]
Changes done as per the suggestions

File: docs/sources/installation/upgrading.md (hunk 2 above)
Comment (NiteshKoushik) [reply docs/sources/installation/upgrading.md]: I have reverted the changes.

**Comment by NiteshKoushik** [comment]
Changes done as per the suggestions

**Review by aknuds1 — APPROVED** [review]

File: docs/sources/installation/upgrading.md [hunk 4]
```diff
@@ -174,7 +174,7 @@ Data sources store passwords and basic auth passwords in secureJsonData encrypte
 will keep working with unencrypted passwords. If you want to migrate to encrypted storage for your existing data sources
 you can do that by:
 
-- For data sources created through UI, you need to go to data source config, re enter the password or basic auth
+- For data sources created through UI, you need to go to data source config, reenter the password or basic auth
```
Comment (achatterjee-grafana) [inline docs/sources/installation/upgrading.md:177]: How about "re-enter"?

File: docs/sources/installation/upgrading.md [hunk 5]
```diff
@@ -189,8 +189,8 @@ this new setting.
 ### Session storage is no longer used
 
 In 6.2 we completely removed the backend session storage since we replaced the previous login session implementation with an auth token.
-If you are using Auth proxy with LDAP an shared cached is used in Grafana so you might want configure [remote_cache] instead. If not
-Grafana will fallback to using the database as an shared cache.
+If you are using Auth proxy with LDAP, a shared cached is used in Grafana, so you might want to configure [remote_cache] instead. If not,
+Grafana will fallback to using the database as a shared cache.
```
Comment (achatterjee-grafana) [inline docs/sources/installation/upgrading.md:193]: Use "fall back"

File: docs/sources/installation/upgrading.md [hunk 6]
```diff
@@ -274,6 +274,6 @@ The following alert notifiers have been updated to support storing their sensiti
 
 For existing alert notification channels, there is no automatic migration of storing sensitive settings encrypted, and they will continue to work as before. Migration must be done manually. Opening a configured alert notification channel in the UI and saving it will store sensitive settings encrypted and at the same time reset the historic unencrypted setting of that alert notification channel in the database.
 
-> Please note that when migrating a notification channel and later downgrading Grafana to an earlier version, the notification channel will not be able to read stored sensitive settings and by that not function as expected.
+> Please note that when migrating a notification channel and later downgrading Grafana to an earlier version, the notification channel will not be able to read stored sensitive settings and as a result not function as expected.
```
Comment (achatterjee-grafana) [inline docs/sources/installation/upgrading.md:277]: Use "and, as a result,"

**Review by achatterjee-grafana — COMMENTED** [review]
Few copy-edit suggestions.

File: docs/sources/installation/upgrading.md (hunk 5 above)
Comment (oddlittlebird) [reply docs/sources/installation/upgrading.md:193]: Agreed. "Fall back" is a verb, "fallback" is a noun. It's similar to "set up" and "setup."

**Comment by NiteshKoushik** [comment]
Hello @aknuds1, any changes needs to be done to merge and close the request. Please let me know.

**Comment by oddlittlebird** [comment]
Please apply @achatterjee-grafana's suggestion.

**Comment by NiteshKoushik** [comment]
Made the changes as per the suggestion

**Comment by NiteshKoushik** [comment]
Hello @oddlittlebird, any changes needs to be done. Please let me know.

File: docs/sources/installation/upgrading.md (hunk 5 above)
Comment (achatterjee-grafana) [reply docs/sources/installation/upgrading.md:193]: Can you change "fallback" to "fall back"? Thank you.

Comment (NiteshKoushik) [reply docs/sources/installation/upgrading.md]: I have made the changes. Thanks for the suggestion @achatterjee-grafana

**Review by achatterjee-grafana — APPROVED** [review]
Thank you!
